phishingpage.py

```diff
@@ -127,8 +127,17 @@ def __init__(self):
                        "Mobile-friendly.")
         office = PhishingTemplate("office365", display_name, description)
 
+        # Browser Plugin Update
+        display_name = "Browser Plugin Update"
+        description = ("A generic browser plugin update template that "  + 
+                       "can be used to serve payloads to targets. "  + 
+                       "Requires prior preparation of payload. Mobile-friendly.")
+        plugin-update = PhishingTemplate("plugin-update", display_name, description)
```

Comment:
`plugin-update` is not a valid Python name for a variable. The script won't even run because of that.